test/integration/ctl/ctl_status_certificate_test.go

```diff
@@ -280,15 +280,15 @@ Issuer:
     type  reason  <unknown>        message
 Secret:
   Name: existing-tls-secret
-  Issuer Country: 
-  Issuer Organisation: 
+  Issuer Country:
```

Comment:
@wallrj I think this change is probably what is causing the test failures.

Reply:
Thanks Tim.  I reverted that whitespace cleanup:
 * https://github.com/cert-manager/cert-manager/compare/91f18e00d8461bf02e58ea82a183afce929531ae..bdd9544959c94a4fdeaf8b249e9d6019da05f3e9